wsclient/src/main/java/org/hpccsystems/ws/client/utils/NamedValueInfo.java

```diff
@@ -0,0 +1,37 @@
+package org.hpccsystems.ws.client.utils;
+
+import org.hpccsystems.ws.client.gen.wsworkunits.v1_69.NamedValue;

```

Comment:
same as above.

Reply:
What's the merge artifact? I don't see it.

Reply:
Ah this is something that should be moved to platform instead of utils.  The only thing left over from the merge is the <<< HEAD that is above (now it looks like it's on line 2155 of HPCCWsWorkUnitsClient.java)

Reply:
OK. I've moved these to the platform package.